includes/class-wc-stripe-customer.php

```diff
@@ -617,7 +645,7 @@ public function delete_id_from_meta() {
 	 *
 	 * @return string ID of the new Customer object.
 	 */
-	private function recreate_customer() {
+	public function recreate_customer() {
```

Comment:
Do we still need to update this function from private to public?
I'm assuming you made this change so that it could be used in the intent controller but it doesn't look like that is the case anymore.

I don't have any issues with this being public, just checking with you if this is intentional :)